templates/macros/https_params.conf

```
@@ -1,3 +1,4 @@
+{% macro https_params(hsts=nginx.hsts, max_age=nginx.hsts.max_age, include_subdomains=nginx.hsts.include_subdomains, preload=nginx.hsts.preload) %}
```

HSTS-related options should use `hsts_` prefix.
```suggestion
{% macro https_params(hsts=nginx.hsts, hsts_max_age=nginx.hsts.max_age, hsts_include_subdomains=nginx.hsts.include_subdomains, hsts_preload=nginx.hsts.preload) %}
```